contentcuration/contentcuration/db/utils.py

```diff
@@ -0,0 +1,33 @@
+from contextlib import contextmanager
+
+from django.db import connections
+from django.db import DEFAULT_DB_ALIAS
+
+
+TEMPORARY_MODELS = {}
+SQL_CREATE_TEMP_TABLE = "CREATE TEMP TABLE %(table)s (%(definition)s) ON COMMIT DROP"
+
+
+@contextmanager
+def temporary_model(model, using=DEFAULT_DB_ALIAS, atomic=True):
```

Comment:
Could you explain how this gets used in a comment?

Reply:
For sure!